### DISCUSSION TITLE
eksctl config files

### DISCUSSION THREAD

**Discussion by rhino007US** [weaveworks/eksctl#4741] [Q&A]
I don't see an example config file for eksctl, nor what it is called.  Here is what happens when i run eksctl:

bash#:   eksctl create cluster -f cluster.yaml
Error: checking AWS STS access – cannot get role ARN for current session: InvalidClientTokenId: The security token included in the request is invalid.
        status code: 403, request id: b16f19e4-ab52-4de9-96f4-37eedb10f4c9

Where does the token come from?  Is ekcctl supposed to use the files in ~/.aws for Account ID and Account key to generate the token?

I have these in place and they work for awscli.

**Comment by cPu1** [comment]
eksctl uses the AWS SDK which uses [this](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials) mechanism for discovering AWS credentials. For most cases, if you've configured your AWS CLI to work with your AWS credentials and are able to successfully run commands like `aws sts get-caller-identity`, eksctl will also work. 

>I don't see an example config file for eksctl, nor what it is called.

You can find example ClusterConfig files in https://github.com/weaveworks/eksctl/tree/main/examples.

**Reply by tsunamaru** [reply]
Is there any way to get an example with all possible values? Scoped examples are good, but I want to see what's ever available.
UPD: nvm, found [it](eksctl.io/usage/schema).
